dependencies.yaml

```diff
@@ -283,8 +282,3 @@ dependencies:
         packages:
           - *dask
           - distributed>=2022.05.2
-  notebooks:
-    common:
-      - output_types: conda
-        packages:
-          - cudf==24.2.*
```

Comment:
There were no references to `cudf` in the notebooks, so I removed this. This dependency was causing devcontainer builds to fail. This has been a pain in the past because it introduced a circular dependency between kvikio and cudf, but it seems safe to remove now.